pkg/odo/cli/cli.go

```diff
@@ -175,6 +175,7 @@ func NewCmdOdo(name, fullName string) *cobra.Command {
 		debug.NewCmdDebug(debug.RecommendedCommandName, util.GetFullName(fullName, debug.RecommendedCommandName)),
 		pipelines.NewCmdComponent(pipelines.RecommendedCommandName, util.GetFullName(fullName, pipelines.RecommendedCommandName)),
 		pipelines.NewCmdInit(pipelines.InitRecommendedCommandName, util.GetFullName(fullName, pipelines.InitRecommendedCommandName)),
+		pipelines.NewCmdAdd(pipelines.AddRecommendedCommandName, util.GetFullName(fullName, pipelines.AddRecommendedCommandName)),
 	)
```

Comment:
Take a look at for example preference.NewCmdPreference above for how to create sub-command.   You want to create a pipelines.NewCmdService().  In pipelines.NewCmdService(), I would add NewCmdAdd().   The command line interface shoould be "odo pipelines services add"

Reply:
changes have been made